### PR TITLE
Flowable exclusions

### PR DESCRIPTION
**Pull request by hottleta**
Modifies code to exclude certain flowable/context combinations using a new CSV input file.

### CODE REVIEW

File: fedelemflowlist/input/FlowExclusions.csv [hunk 1]
```diff
@@ -0,0 +1,45 @@
+Flowable,Context
+"Water, fresh",resource/water/brackish water body
+"Water, fresh",resource/water/brackish water body/lake
+"Water, fresh",resource/water/brackish water body/lake/rural
+"Water, fresh",resource/water/brackish water body/lake/urban
```
Comment (bl-young) [inline fedelemflowlist/input/FlowExclusions.csv:5]: is there a brackish water body/river option?

Comment (hottleta) [reply fedelemflowlist/input/FlowExclusions.csv:5]: no

**Comment by bl-young** [comment]
@WesIngwersen  this is to solve #106, do you have any concerns with this approach?

**Review by WesIngwersen — APPROVED** [review]
Approve - a good approach to take for now

**Pull request merged**